### ISSUE TITLE
QNN support

### ISSUE THREAD

**Issue by lutzroeder** [lutzroeder/netron#1283]
**Documentation:**
https://docs.qualcomm.com/bundle/publicresource/topics/80-63442-50/converters.html

**Examples:**
[mobilenetv2-12_net.json](https://github.com/user-attachments/files/15507409/mobilenetv2-12_net.json)

**Comment by MeghnaNatraj** [comment]
**UPDATE: This feature is still missing in Netron.**

TL;DR: We want to visualize the QNN model (serialized.bin file) and not the QNN metadata (.json file). Is it possible to add support to visualize the **serialized.bin** file? Currently, I get the error "File contains undocumented QNN serialized context." when I try to open the file in Netron.

Reference:
There are 4 files generated when converting a model from TFLite to QNN. Let us take the Resnet model as an example. You can refer to the files here ([TFLite and QNN models](https://drive.google.com/drive/folders/1zAwPDfzHe2_II2x_pLIqUCzAjm0LNiG3?usp=sharing)):

TFLite:
1. resnet.tflite: Can be visualized on Netron

QNN files:
1. **[resnet.serialized.bin](https://drive.google.com/file/d/1J7JRSnQ0-aYrlCrk7EDfCOatLFhQrhmU/view?usp=drive_link) (actual model): Cannot run on Netron**
2. resnet.json: (not the model, just a JSON with a general model structure, fewer ops, etc): Can be visualized on Netron
3. resnet.cpp: (not relevant)
4. resnet.bin: (not relevant)

**Comment by lutzroeder** [comment]
@MeghnaNatraj can you provide the documentation for content  the `.serialized.bin` format and send a pull request adding support.

**Comment by MeghnaNatraj** [comment]
@lutzroeder I've asked Qualcomm for details on the `.serialized.bin` format and will post an update here when I hear back.
